src/transformers/models/mamba2/modeling_mamba2.py

```diff
@@ -1838,23 +1838,16 @@ def _update_causal_mask(self, attention_mask, inputs_embeds, past_key_values, ou
     MAMBA2_START_DOCSTRING,
 )
 class Mamba2ForCausalLM(Mamba2PreTrainedModel):
-    _tied_weights_keys = ["lm_head.weight", "backbone.embeddings.weight"]
+    _tied_weights_keys = ["lm_head.weight"]
```

Comment:
Tied weights have been done like this anywhere else, so hopefully this should be good.